### PR TITLE
show/hide dotfiles

### PR DESCRIPTION
**Pull request by brianmcmichael**
This works by setting a cookie on the checkbox state change and then if the "Show dotfiles" box is unchecked, the app deletes the elements marked with the `dotfile` class.

By deleting the elements instead of hiding them we don't break the arrow/pgup/pgdn/selection stuff.

### CODE REVIEW

**Comment by ericfranz** [comment]
The added javascript below `CloudCmd('{{ prefix }}');` on the `index.html` page potentially creates a race condition. Better to bind to the events before they could be triggered.

I would move all of the code to a new `lib/client/ood.js` and then we would have this in the index.html:

``` html
        </div>


<script src="{{ prefix }}/lib/client/ood_tree.js"></script>
<script src="{{ prefix }}/lib/client/ood.js"></script>
<script>
// Event handler activates on page load
jQuery12('body').on('panel_initialized', function(e){
    initTree("{{ treeroot }}");
    document.getElementById("checkbox-dotfiles").checked = ( getCookie('dotfiles') === 'true' );
    dotfiles();
});
</script>


<script src="{{ prefix }}/cloudcmd.js"></script>
<script>
CloudCmd('{{ prefix }}');
</script>

</head>
</body>
</html>
```
- Notice that this block must be in the html since the `{{ treeroot }}` is being rendered.
- Also notice that I did not preserve the HTML indentation. The scripts are not part of the HTML and indenting them alongside the HTML doesn't help the inline script readability. Better to keep scripts without the indentation.

**Comment by brianmcmichael** [comment]
Current code formatting was all based on original codebase. I don't think pushing it all up against the gutter makes it more readable, but say the word and I'll do it.

I've been putting the script code here for now for convenience and since there are no caching issues that require a restart, although directly calling the files would work fine.

**Comment by ericfranz** [comment]
Since we have a cache busting solution that adds a header to all requests, the css in a separate file is cache busted as it is inline. Please remove the indentation on the main html page from the script tags at the bottom for the js that is left as inline scripts.

File: css/ood_styles.css [hunk 1]
```diff
@@ -168,8 +167,7 @@ span.owner, span.mode {
 
 .panel-btns {
     position: absolute;
-    top: -40px;
-    right: 0px;
+    right: 30px;
```
Comment (ericfranz) [inline css/ood_styles.css:170]: adding `top: 10px` makes it look nice I think - tiny space above (only tested in chrome though)

Comment (ericfranz) [inline css/ood_styles.css:170]: ha `padding-top: 10px;` achieves the same effect or  `margin-top: 10px;`

**Pull request merged**